open-irc-links/README.md

```diff
@@ -0,0 +1,14 @@
+# open-irc-links
+
+
+## What it does
+
+This add-on sets the default client for opening IRC links using protocol_handlers.
+
+Whenever a link using the IRC protocol is clicked, the link is opened in the URI defined in the add-on's manifest.
+In this example, all IRC protocol links are opened in mibbit.com.  
+For example, click [here](irc://irc.freenode.net/drupal) and the protocol handler opens the Drupal channel in mibbit.com.
```

Comment:
Please expose the URL so the reader can see what they clicking. Also, we should probably point out they need to install the extension first:
"For example, once you've installed the extension, open this link [irc://irc.freenode.net/drupal](irc://irc.freenode.net/drupal) and the protocol handler opens the Drupal channel in mibbit.com."